x-pack/plugins/ml/server/routes/results_service.ts

```diff
@@ -265,4 +283,66 @@ export function resultsServiceRoutes({ router, mlLicense }: RouteInitialization)
       }
     })
   );
+
+  /**
+   * @apiGroup ResultsService
+   *
+   * @api {get} /api/ml/results/:jobId/categorizer_stats
+   * @apiName GetCategorizerStats
```

Comment:
The two new API roues you added here should also go into `ml/server/routes/apidoc.json` so that they get added to the docs.

Reply:
Added the routes here https://github.com/elastic/kibana/pull/75061/commits/981e6e48a26fa37d6466fc7466d191daa873aef2